src/measure_theory/set_integral.lean

```diff
@@ -763,6 +763,11 @@ begin
   all_goals { assumption }
 end
 
+lemma apply_integral {H : Type*} [normed_group H] [normed_space ℝ H]
```

Comment:
`integral_apply`, maybe? Also, the right hand side of the lemma seems more complicated to me than the left hand side, so I'd probably swap them.